tailcfg/derpmap.go

```diff
@@ -29,6 +36,23 @@ func (m *DERPMap) RegionIDs() []int {
 	return ret
 }
 
+// DERPHomeParams contains parameters from the server related to selecting a
+// DERP home region (sometimes referred to as the "preferred DERP").
+type DERPHomeParams struct {
+	// RegionWeight scales latencies of DERP regions by a given scaling
+	// factor when determining which region to use as the home
+	// ("preferred") DERP.
+	//
+	// If a region is not present in this map, it is treated as having a
+	// weight of 1.0.
+	//
+	// Weights should not be 0 or negative; such weights will be ignored.
+	//
+	// A nil map means no change from the previous value (if any); an empty
+	// non-nil map can be sent to reset all weights back to 1.0.
```

Comment:
i got confused a bit reading this and the code, code looks fine but weight implies that if the value increases, the preference increases. but its the opposite here.
What about `RegionLatencyMultiplier` or something that implies that lower values should be assigned to regions that are preferred

Reply:
kinda a mouthful, that name

is `RegionScale` enough to not sound bigger-is-better to you?

Reply:
`RegionScore` with docs that call out 0<x<1 makes it preferred and >1 penalizes?

Reply:
Renamed to `RegionScore` and added docs on the semantics.